lib/commands/create-project.ts

```diff
@@ -27,12 +29,20 @@ export class CreateProjectCommand implements ICommand {
 		private $errors: IErrors,
 		private $logger: ILogger,
 		private $projectNameValidator: IProjectNameValidator,
-		private $options: ICommonOptions) { }
+		private $options: IOptions) { }
 
 	public enableHooks = false;
 
 	execute(args: string[]): IFuture<void> {
 		return (() => {
+			if (this.$options.ng && this.$options.template) {
+				this.$errors.fail("You cannot use --ng and --template simultaneously.");
+			}
+
+			if (this.$options.ng) {
+				this.$options.template = constants.ANGULAR_NAME;
```

Comment:
instead of setting `this.$options.template` you can use something like:
```TypeScript
let selectedTemplate = this.$options.ng ? constants.ANGULARE_NAME : this.$options.template;
this.$projectService.createProject(args[0], selectedTemplate).wait();
```